Tests/Functional/Doctrine/Phpcr/SitemapDocumentProviderTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Tests\Functional\Doctrine\Phpcr;
+
+use Symfony\Cmf\Bundle\SeoBundle\Doctrine\Phpcr\SitemapDocumentProvider;
+use Symfony\Cmf\Component\Testing\Functional\BaseTestCase;
+
+/**
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class SitemapDocumentProviderTest extends BaseTestCase
+{
+    /**
+     * @var SitemapDocumentProvider
+     */
+    private $documentProvider;
+
+    public function setUp()
+    {
+        $this->db('PHPCR')->createTestNode();
+        $this->dm = $this->db('PHPCR')->getOm();
+        $this->base = $this->dm->find(null, '/test');
+        $this->db('PHPCR')->loadFixtures(array(
+            'Symfony\Cmf\Bundle\SeoBundle\Tests\Resources\DataFixtures\Phpcr\LoadSitemapData',
+        ));
+
+        $this->documentProvider = new SitemapDocumentProvider($this->dm);
+    }
+
+    public function testDocumentOrder()
+    {
+        $documents = $this->documentProvider->load('default');
+
+        $paths = array();
+        foreach ($documents as $document) {
+            $paths[] = $document->getId();
+        }
+
+        $this->assertEquals(
```

Comment:
@dbu  this a a hard coded test on the sorting order of the document's path i get by the provider and its query.